documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -514,7 +514,8 @@ Setting an appropriate memory request can prevent the following:
 If `-Xms` is set to `-Xmx`, the container will crash immediately; if not, the container will crash at a later time.
 
 In this example, the JVM uses 2 GiB (=2,147,483,648 bytes) for its heap.
-Its total memory usage is approximately 8GiB.
+Its total memory usage is approximately 8 GiB.
```

Comment:
I would leave this out. I don't think we have any data to back this number.

Reply:
removed